File: Models/Equipe.cs
namespace SuperBowlWeb.Models
{
    public class Equipe
    {
        public int Id { get; set; }

        public string Nom { get; set; }
        public int Cote { get; set; }

        public int? PaysId { get; set; }

        public Pays Pays { get; set; }
        public string URLlogo { get; set; }

        public ICollection<Joueur>? Joueurs{ get; set; }
    }
}

File: frontend/src/Modele/Equipe.ts
import Joueur from "./Joueur";
import Pays from "./Pays";

export default interface Equipe{
    id:number,
    nom:string,
    cote:number,
    paysId:number,
    urLlogo:string,
    pays:Pays,
    joueurs:Joueur[]
}